lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -611,9 +612,9 @@ def test_submission_history_accepts_valid_ids(self):
         assert self.client.login(username=admin.username, password='test')
 
         url = reverse('submission_history', kwargs={
-            'course_id': str(self.course_key),
-            'student_username': 'dummy',
-            'location': str(self.problem.location),
+            'course_id': six.text_type(self.course_key),
+            'learner_identifier': 'dummy',
+            'location': six.text_type(self.problem.location),
```

Comment:
with the upgrade to python3 I don't think these `six` calls are needed for Python 2 and 3 Compatibility. 
this can be simply `str(self.course_key)`